catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -151,6 +151,13 @@
             default="^sortOrder$,^sortField$,^cql$,^polling$,^cached$"
             required="false"/>
 
+        <AD id="attributeDescriptions"
+            name="Attribute Descriptions"
+            description="List of friendly attribute descriptions. Example 'checksum-algorithm=Method for generating a small-sized datum from a block of digital data for the purpose of detecting errors'"
```

Comment:
We may want to emphasize that there must be an `=` sign for the attribute=attribute description